app/decorators/decidim/invite_user_decorator.rb

```diff
@@ -19,8 +20,8 @@ def invite_user
       email: form.email.downcase,
       nickname: Decidim::UserBaseEntity.nicknamize(form.name, organization: form.organization),
       organization: form.organization,
-      admin: form.role == "admin",
-      roles: form.role == "admin" ? [] : [form.role].compact
+      admin: form.role == 'admin',
+      roles: form.role == 'admin' ? [] : [form.role].compact
```

Comment:
maybe add a private method `admin?` to avoid repeating `form.role == 'admin'` ? (totally optional)

Reply:
@aitorlb It is not being repeated, lines starting with - are removals and lines starting with + are additions. Anyway i'll go for the refactor :+1:

Reply:
I meant:
```ruby
...
  admin: admin?,
  roles: admin? ? [] : [form.role].compact
...
private

def admin?
  form.role == 'admin' 
end
```

Reply:
done

Reply:
Fully done :D